index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alazar Esheetu</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  </head>
+  <body>
+    <div class="contanier">
+      <nav class="nav-contanier">
+        <div class="logo">Alazar Eshetu</div>
+        <div>

```

Comment:
- [ ] This should be your page's toolbar, and you should use proper HTML tags to wrap the elements for better semantics and increase the SEO of your page. Kindly wrap the nav with a header tag instead of `div.` Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation.

- [ ] Kindly wrap the logo in an anchor tag to be linked in some important section of your portfolio and a convention. :smiley_cat: 


Example
```
<header>
     <nav> 
           <a href =" " > Alex </a>
           <img src=" ./ " alt=" mobile menu />
    </nav>
</header>
```